papers_list.md

```diff
@@ -25,3 +25,4 @@
 | [DeepChain: Auditable and Privacy-Preserving Deep Learning with Blockchain-based Incentive](https://eprint.iacr.org/2018/679.pdf) | > Recently, privacy-preserving deep learning has drawn tremendous attention from information security community, in which neither training data nor the training model is expected to be exposed. |
 | [Decoding individual differences in STEM learning from functional MRI data](https://www.nature.com/articles/s41467-019-10053-y) | > Here, we investigated whether patterns of brain activity collected during a concept knowledge task could be used to compute a neural ‘score’ to complement traditional scores of an individual’s conceptual understanding. |
 | [Generative Adversarial Nets](https://papers.nips.cc/paper/5423-generative-adversarial-nets.pdf) | > We propose a new framework for estimating generative models via an adversarial process, in which we simultaneously train two models: a generative model G that captures the data distribution, and a discriminative model D that estimates the probability that a sample came from the training data rather than G. |
+| [A Unified Approach to Interpreting ModelPredictions](http://papers.nips.cc/paper/7062-a-unified-approach-to-interpreting-model-predictions.pdf) | Understanding why a model makes a certain prediction can be as crucial as theprediction’s accuracy in many applications. However, the highest accuracy for largemodern datasets is often achieved by complex models that even experts struggle tointerpret, such as ensemble or deep learning models, creating a tension between accuracy and interpretability|
```

Comment:
Small changes need to be made:

- space needed between `ModelPredictions`
- following the same format, a `>` is missing for piece of abstract
- space needed between `theprediction’s`
- space needed between `largemodern`
- space needed between `ltointerpret`
- final dot missing at final of piece of abstract